db/fixtures/ae_datastore/ManageIQ/Cloud/Orchestration/Operations/Methods.class/__methods__/available_vdc_networks.rb

```diff
@@ -0,0 +1,47 @@
+class AvailableVdcNetworks
+  def initialize(handle = $evm)
+    @handle = handle
+  end
+
+  def main
+    fill_dialog_field(fetch_list_data)
+  end
+
+  def fetch_list_data
+    service = @handle.root.attributes["service_template"] || @handle.root.attributes["service"]
+    vdc_networks = service.try(:orchestration_manager).try(:cloud_networks).try(:select) do |net|
```

Comment:
The chain of tries may resolve into `nil`, so the `do` block will fail. Please refactor.